pages/index.ts

```diff
@@ -1 +1 @@
-export { default } from 'components/pages/index';
+export { default } from 'components/pages/home';
```

Comment:
I think it's better to name it as home, because usually `index` means some access point  to whole folder (pages/index it's kinda exception for next.js)